opencti-platform/opencti-graphql/config/schema/opencti.graphql

```diff
@@ -1054,7 +1054,7 @@ input RetentionRuleAddInput {
   name: String! @constraint(minLength:2, format: "not-blank")
   filters: String!
   max_retention: Int! @constraint(min: 1)
-  retention_unit: RetentionUnit
+  retention_unit: RetentionUnit!
```

Comment:
having the value mandatory in type `RetentionRule` I agree, if this is indeed a value always present when querying a rule.

But having the value mandatory in the input, is a **breaking change**.
If the API accepts nothing (and defaults to 'days', it's fine)

Reply:
Indeed ! Fixed :)
I've also made filters not mandatory in RetentionRuleAddInput to facilitate creation via the API (user can write no filters instead of an empty filter groups)

Reply:
I don't understand, it's not fixed? The fix should be to removed this "!" no? Or I am misunderstanding?

Reply:
to remove the '!' for RetentionRuleAddInput indeed, but not for RetentionRule